runtime/compiler/p/runtime/PPCRelocationTarget.cpp

```diff
@@ -202,6 +202,10 @@ TR_PPC32RelocationTarget::storeRelativeAddressSequence(uint8_t *address, uint8_t
 void
 TR_PPC64RelocationTarget::storeAddressSequence(uint8_t *address, uint8_t *reloLocation, uint32_t seqNumber)
    {
+   //Contingious Word
+   if(seqNumber == 7)
```

Comment:
Probably cleaner to change this to `seqNumber == fixedSequence7`. While you're at it, could you also update the case statements below to also refer to the enum?

Reply:
Please also follow the coding conventions: space between if and (